src/Infrastructure/KafkaJanitor.RestApi/Startup.cs

```diff
@@ -61,7 +61,8 @@ public void ConfigureServices(IServiceCollection services)
             // Enablers
             var shouldStartMetricHostedService = Configuration["KAFKAJANITOR_START_METRIC_SERVER"] != "false";
             services.AddMetrics(shouldStartMetricHostedService);
-            services.ConfigureHealthChecks();
```

Comment:
Parse string value to boolean and make a proper comparrison? What happens right now if cfg value == False and code is configured to be case sensitive